<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TS_RegressionTest" time="179" tests="1" failures="2" errors="0">
   <testsuite name="TS_RegressionTest" tests="1" failures="2" errors="0" time="179" timestamp="2018-07-02 09:56:39" hostname="haimnguyen - 3KR8462.mshome.net" id="Test Suites/TS_RegressionTest">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637"/>
         <property name="logFiles" value="c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\execution0.log"/>
         <property name="attachments" value="c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500272872.png, c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500378021.png, c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500378284.png"/>
         <property name="hostName" value="haimnguyen - 3KR8462.mshome.net"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="5.4.2.1"/>
         <property name="browser" value="Chrome 67.0.3396.99"/>
         <property name="hostAddress" value="172.22.134.33"/>
         <property name="sessionId" value="706e20e983a63f78c29b05cbf12ad3c5"/>
         <property name="seleniumVersion" value="3.7.1"/>
         <property name="proxyInformation" value="ProxyInformation{proxyOption=NO_PROXY, proxyServerType=HTTP, password=, proxyServerAddress=, proxyServerPort=0}"/>
         <property name="platform" value="Windows 10"/>
      </properties>
      <testcase name="Test Cases/Main Test Cases/TC1_Verify Successful Login" classname="Test Cases/Main Test Cases/TC1_Verify Successful Login" status="PASSED">
         <system-out><![CDATA[2018-07-02 09:56:39 - [TEST_CASE][PASSED] - Test Cases/Main Test Cases/TC1_Verify Successful Login: Test Cases/Main Test Cases/TC1_Verify Successful Login

2018-07-02 09:56:40 - [TEST_STEP][PASSED] - comment: Story: Login to CURA system

2018-07-02 09:56:40 - [TEST_STEP][PASSED] - comment: Given that the user has the valid login information

2018-07-02 09:56:40 - [TEST_STEP][PASSED] - openBrowser: Browser is opened with url: 'http://demoaut.katalon.com'

2018-07-02 09:56:46 - [MESSAGE][PASSED] - Browser is opened with url: 'http://demoaut.katalon.com'

2018-07-02 09:56:46 - [TEST_STEP][PASSED] - click: Object: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' is clicked on

2018-07-02 09:56:47 - [MESSAGE][PASSED] - Object: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' is clicked on

2018-07-02 09:56:47 - [TEST_STEP][PASSED] - setText: Text 'John Doe' is set on object 'Object Repository/Page_Login/txt_UserName'

2018-07-02 09:56:47 - [MESSAGE][PASSED] - Text 'John Doe' is set on object 'Object Repository/Page_Login/txt_UserName'

2018-07-02 09:56:47 - [TEST_STEP][PASSED] - setText: Text 'ThisIsNotAPassword' is set on object 'Object Repository/Page_Login/txt_Password'

2018-07-02 09:56:47 - [MESSAGE][PASSED] - Text 'ThisIsNotAPassword' is set on object 'Object Repository/Page_Login/txt_Password'

2018-07-02 09:56:47 - [TEST_STEP][PASSED] - comment: When he logins to CURA system

2018-07-02 09:56:47 - [TEST_STEP][PASSED] - click: Object: 'Object Repository/Page_Login/btn_Login' is clicked on

2018-07-02 09:56:48 - [MESSAGE][PASSED] - Object: 'Object Repository/Page_Login/btn_Login' is clicked on

2018-07-02 09:56:48 - [TEST_STEP][PASSED] - comment: Then he should be able to login successfully

2018-07-02 09:56:48 - [TEST_STEP][PASSED] - Statement - landingPage = com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.verifyElementPresent(com.kms.katalon.core.testobject.ObjectRepository.findTestObject(Page_CuraAppointment/div_Appointment), G_Timeout): Object 'Object Repository/Page_CuraAppointment/div_Appointment' is present

2018-07-02 09:56:48 - [MESSAGE][PASSED] - Object 'Object Repository/Page_CuraAppointment/div_Appointment' is present

2018-07-02 09:56:48 - [TEST_STEP][PASSED] - closeBrowser: Browser is closed

2018-07-02 09:56:49 - [MESSAGE][PASSED] - Browser is closed]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/Main Test Cases/TC2_Verify Successful Appointment" classname="Test Cases/Main Test Cases/TC2_Verify Successful Appointment" status="FAILED">
         <failure type="FAILED" message="Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)"/>
         <system-out><![CDATA[2018-07-02 09:56:49 - [TEST_CASE][FAILED] - Test Cases/Main Test Cases/TC2_Verify Successful Appointment: Test Cases/Main Test Cases/TC2_Verify Successful Appointment FAILED because (of) Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)

2018-07-02 09:56:49 - [TEST_STEP][PASSED] - comment: Story: Book an appointment

2018-07-02 09:56:49 - [TEST_STEP][PASSED] - comment: Given that the user has logged into their account

2018-07-02 09:56:49 - [TEST_STEP][PASSED] - openBrowser: Browser is opened with url: 'http://demoaut.katalon.com'

2018-07-02 09:57:21 - [MESSAGE][PASSED] - Browser is opened with url: 'http://demoaut.katalon.com'

2018-07-02 09:57:21 - [TEST_STEP][FAILED] - callTestCase: Unable to call Test Case with id 'Test Cases/Common Test Cases/Login' (Root cause: Call Test Case 'Test Cases/Common Test Cases/Login' failed)

2018-07-02 09:57:22 - [TEST_CASE][FAILED] - Test Cases/Common Test Cases/Login: Test Cases/Common Test Cases/Login FAILED because (of) Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)

2018-07-02 09:57:22 - [TEST_STEP][FAILED] - click: Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)

2018-07-02 09:57:53 - [MESSAGE][FAILED] - Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)
[[ATTACHMENT|c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500272872.png]]

2018-07-02 09:57:53 - [MESSAGE][FAILED] - Unable to call Test Case with id 'Test Cases/Common Test Cases/Login' (Root cause: Call Test Case 'Test Cases/Common Test Cases/Login' failed)]]></system-out>
         <system-err><![CDATA[2018-07-02 09:56:49 - [TEST_CASE][FAILED] - Test Cases/Main Test Cases/TC2_Verify Successful Appointment: Test Cases/Main Test Cases/TC2_Verify Successful Appointment FAILED because (of) Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)]]></system-err>
      </testcase>
      <testcase name="Test Cases/Main Test Cases/A failed test case" classname="Test Cases/Main Test Cases/A failed test case" status="FAILED">
         <failure type="FAILED" message="Unable to verify alert present (Root cause: No alert found after 100 second(s).)"/>
         <system-out><![CDATA[2018-07-02 09:57:53 - [TEST_CASE][FAILED] - Test Cases/Main Test Cases/A failed test case: Test Cases/Main Test Cases/A failed test case FAILED because (of) Unable to verify alert present (Root cause: No alert found after 100 second(s).)

2018-07-02 09:57:53 - [TEST_STEP][PASSED] - openBrowser: Browser is opened with url: 'https://google.com'

2018-07-02 09:57:53 - [MESSAGE][WARNING] - A browser is already opened. Closing browser and opening a new one

2018-07-02 09:57:57 - [MESSAGE][PASSED] - Browser is opened with url: 'https://google.com'

2018-07-02 09:57:57 - [TEST_STEP][FAILED] - verifyAlertPresent: Unable to verify alert present (Root cause: No alert found after 100 second(s).)

2018-07-02 09:59:38 - [MESSAGE][FAILED] - No alert found after 100 second(s).
[[ATTACHMENT|c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500378021.png]]

2018-07-02 09:59:38 - [MESSAGE][FAILED] - Unable to verify alert present (Root cause: No alert found after 100 second(s).)
[[ATTACHMENT|c:\\data\\katalon-projects\\python-test\\Reports\\TS_RegressionTest\\20180702_095637\\1530500378284.png]]]]></system-out>
         <system-err><![CDATA[2018-07-02 09:57:53 - [TEST_CASE][FAILED] - Test Cases/Main Test Cases/A failed test case: Test Cases/Main Test Cases/A failed test case FAILED because (of) Unable to verify alert present (Root cause: No alert found after 100 second(s).)]]></system-err>
      </testcase>
      <system-out><![CDATA[2018-07-02 09:56:39 - [TEST_SUITE][FAILED] - TS_RegressionTest: Test Cases/Main Test Cases/TC2_Verify Successful Appointment FAILED because (of) Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)]]></system-out>
      <system-err><![CDATA[2018-07-02 09:56:39 - [TEST_SUITE][FAILED] - TS_RegressionTest: Test Cases/Main Test Cases/TC2_Verify Successful Appointment FAILED because (of) Unable to click on object 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' (Root cause: com.kms.katalon.core.webui.exception.WebElementNotFoundException: Web element with id: 'Object Repository/Page_CuraHomepage/btn_MakeAppointment' located by 'By.xpath: //*[@id = 'btn-make-appointment']' not found)]]></system-err>
   </testsuite>
</testsuites>
